relaybot.py

```diff
@@ -52,7 +53,15 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        optionAsBoolean = { "": False, "false": False, "no": False, "true": True, "yes": True }

```

Comment:
I wanted it to come out as `False` if the option's not set, so as not to disrupt when people haven't configured it. Should that be `None: False`?